### DISCUSSION TITLE
My Crazyflie 2.1+ won't fly

### DISCUSSION THREAD

**Discussion by prajwas2004** [bitcraze/discussions#1561] [Q&A]
I just got this today.  I am trying to fly this thing.  It won't take off.  I made sure the propellers are on correctly and they are all rotating. 
Any other issues?

https://github.com/user-attachments/assets/2b9c93f7-c100-4b89-8a1a-095b75c0d93a

![Screenshot_20240922_025223_Crazyflie Client](https://github.com/user-attachments/assets/c44fe17e-2d8b-4710-a7d4-3fa5bc3da691)

**Comment by prajwas2004** [comment]
Added video
Uploading VID-20240922-WA0003.mp4…

**Comment by ToveRumar** [comment]
Hi!

Thank you for your question. 

Its a bit hard to see from the video, but it looks like they might be attached upside down? So that the concave side is up, the convex side should be up. 

Do you have any decks connected to it? 

I would suggest downloading the client (https://github.com/bitcraze/crazyflie-clients-python) and try connecting (and maybe flying) with it. There you will get some more information, on the battery charge level as well as full logs. This might give a bit more data on the issue. 

Please come back with some logs if possible, if the propellers are not the issue.

**Comment by prajwas2004** [comment]
![Screenshot_20240926_231013_Gallery](https://github.com/user-attachments/assets/1f4de21e-d796-4143-933a-6597b2ffdb30)
No decks are connected. 
The propellers are connected so that the number markings are underneath and not above. 

I have tried both ways and am not getting lift. Have tried all combinations. 

Charge levels. I have charged it for an hour. Both Blue lights are steady(not blinking).  

My Asus Rog laptop won't connect to the Bluetooth on this so connecting through python option is not there. 

I do not intend to debug a new item. There is no additional decks here.

**Reply by ToveRumar** [reply]
Hi! 

Thank you for the picture and the explanation of the propeller setup. 

The number markings should be on top. So just flip your propellers and it should be able to lift. 

Hope this fixes your issue!
